lib/modules/dosomething/dosomething_global/dosomething_global.install

```diff
@@ -67,7 +67,7 @@ function dosomething_global_update_7003(&$sandbox) {
 /**
  * Uninstall some unused modules.
  */
-function dosomething_global_update_7004() {
+function dosomething_global_update_7005() {
```

Comment:
There is one for `7003` already, which is why I originally did `7004`, but now that `7004` has already run on Staging, I had to bump it to `7005` to get it to re-run now that the dependency has been removed. Make sense? Thoughts?